app/blog/2017-10-18-upcoming-changes.md

```diff
@@ -0,0 +1,146 @@
+---
+title:  "Changes to shadow DOM v0 and HTML imports"
+---
+
+
+In the near future, Chrome is rolling out two changes that may affect existing Polymer 1.x and 2.x websites:
+
+
+*   Shadow-piercing selectors (`/deep/` and `::shadow`). Chrome 63 will disable the shadow-piercing selectors in style sheets. This change only affects sites using Polymer 1.x that are using native shadow DOM.
+
+*   HTML imports styling the main document. Chrome 65 will make a change to the behavior of HTML imports. Currently, styles from an HTML import are automatically applied to the main document. This behavior is going away in Chrome 65. This change affects sites using either Polymer 1.x or Polymer 2.x. 
+
+The following sections describe these changes in more detail.
+
+## Shadow-piercing selectors
+
+From Chrome 63 on, the `/deep/` combinator and `::shadow` pseudo class selector won't pierce shadow roots when used in style sheets:
+
+
+
+*   The `/deep/` combinator acts like the descendant combinator (in other words `html /deep/ div` is treated the same as `html div`). This prevents some global style sheets from breaking entirely.
+*   The `::shadow` pseudo class selector doesn't match any elements.
+
+Note `/deep/` and `::shadow` still function when used from JavaScript APIs like `querySelector` and `querySelectorAll`. 
+
+We first posted about [moving away from /deep/](https://www.polymer-project.org/blog/2015-12-01-deprecating-deep.html) in December 2015, so hopefully this is not news to most people. This change only affects Polymer 1.x and web components v0; the web components v1 specs used in Polymer 2.x don't support `/deep/` or `::shadow `at all.
+
+This change is already in Chrome Canary, and will roll out to the stable channel at the beginning of December. Now is the time to test your sites and make sure that they work in Canary.
+
+If you don't have it, [get Chrome Canary here](https://www.google.com/chrome/browser/canary.html).
+
+There are two possible ways to fix this issue:
+
+
+
+*   Remove the use of `/deep/` and `::shadow` (recommended).
+
+*   Use shady DOM instead of native shadow DOM.
+
+
+### Removing /deep/ and ::shadow
+
+The best fix is to eliminate any remaining use of `/deep/` and `::shadow` in your sites. The `::shadow` pseudo class has been fairly lightly used, but `/deep/` was used in the past for theming.
+
+For example, there was a previous version of the iron-flex-layout import that used `/deep/`. Search for the following string:
```

Comment:
Suggest using backticks around iron-flex-layout

>For example, there was a previous version of the `iron-flex-layout` import that 

Instead of 

>For example, there was a previous version of the iron-flex-layout import that